### DISCUSSION TITLE
Post your general NP questions here

### DISCUSSION THREAD

**Discussion by mdotstrange** [mdotstrange/NightmarePuppeteerPublic#408] [General]
If you have any questions feel free to post them here-

**Comment by RayR10** [comment]
Question about customizing other bodies besides Skinman template.  Are there other templates?

Hi.  I used that Skinman body and template to make a quick custom character.  And it turned out okay.  Except I hate how you can see the back of the head and the face is really just a mask.  I'd rather the custom skin cover the entire head.  Also... that Skinman model is kind of chunky.  
![Screenshot 2024-06-23 161013](https://github.com/mdotstrange/NightmarePuppeteerPublic/assets/172993710/defed61c-d1b7-4ab1-8452-f08d9ca0a45b)
![Screenshot 2024-06-19 203633](https://github.com/mdotstrange/NightmarePuppeteerPublic/assets/172993710/748af409-729e-410e-b447-dc00e066c453)
So... I saw in your videos that you added an option to load a Custom Flavor onto any character... and that's cool... however... how do you know how to make the skin for each model?  Just for the heck of it, I put my dude in socks and underwear skin onto another character and you can see it doesn't work.  You can see it came out disastrous....

![Screenshot 2024-06-23 160919](https://github.com/mdotstrange/NightmarePuppeteerPublic/assets/172993710/ea4ceff1-69b2-48a6-835c-bf705c3dcbe4)

Are all the other character's templates found somewhere to download and use?  Is there a certain height/width we should use?  Thank you for any help you can provide.

**Reply by RayR10** [reply]
I tried making a multi-colored grid with numbers to see if I could figure out what this is doing... but I'm stumped.  It looks like, no matter what size you do, it cuts it off at the waist and then repeats the pattern again.  And then it seems to make separate arms/gloves. 
![image](https://github.com/mdotstrange/NightmarePuppeteerPublic/assets/172993710/67b576ff-338b-4090-8f84-02544a78f04d)
